kaupunginnaiset/alman-akka
# Add logic for events adding

## Purpose

Add button for adding dummy event to database. Using now only dummy events, but same logic can be used when adding the event form.

At localhost the event is added to emulator environment and update triggered to dev-time backend API. In production environment the event addition triggers fetch gh action workflow.

## Screenshots

<img width="639" alt="image" src="https://user-images.githubusercontent.com/29113682/165889248-c220cb40-8a9a-4f46-a9ec-7a040f29193c.png">

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/29113682/165889274-301e5982-98f9-49a5-9b51-2433544b86fa.png">

<img width="1036" alt="image" src="https://user-images.githubusercontent.com/29113682/165889296-df59a825-2cb9-40f4-92b1-b57c970d20f4.png">



## Related Issues

Closes #38 

## How to test

* Start emulator with `yarn tools:emulator`
* Login
* Add event with "Lisää tapahtuma"-button
* Refresh page and see that event is listed to "Omat tapahtumat"
